libs/angular/core/package.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "@studiohyperdrive/ngx-core",
+	"version": "19.0.0",
+  "description": "A library of core functionality used by the Studio Hyperdrive NGX packages.",
+  "keywords": [
+    "angular"
```

Comment:
Perhaps `Studio Hyperdrive`, `StudioHyperdrive`, and `ngx-tools` could be additional keywords?

Reply:
Additional thought: maybe `ngx-tools` can be a keyword of every package as to connect them more easily through npm?